<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wallbreaker</class>
 <widget class="QDialog" name="Wallbreaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wallbreaker</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>查询结果</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="txtSearchData">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>操作</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2" rowspan="4">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnClassSearch">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>classsearch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClassDump">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>classdump</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnObjectSearch">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>objectsearch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnObjectDump">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>objectdump</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClearUI">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>地址：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtAddress"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtClassName"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>类名：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QListWidget" name="listClasses"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
